src/content-cli-config.ts

```diff
@@ -70,13 +70,30 @@ export class Config {
 
         return program;
     }
+
+    public static diff(program: CommanderStatic): CommanderStatic {
+        program
+            .command("diff")
+            .description("Command to diff configs of packages")
+            .option("-p, --profile <profile>", "Profile of the team/realm which you want to use to diff the packages with")
+            .option("--hasChanges", "Flag to return only the information if the package has changes without the actual changes")
+            .option("--json", "Return the response as a JSON file")
+            .requiredOption("-f --file <file>", "Exported packages file (relative path)")
```

Comment:
I think this should work even if the passed file is absolute, right?

Reply:
Yes. It works with both actually. I'll update it.

Reply:
Changed it [here](https://github.com/celonis/content-cli/pull/184/commits/04bb9d7f9c9017295c517198c5d4f7c591823085).